add-ons/strimzi/strimzi.addon

```diff
@@ -0,0 +1,26 @@
+# Name: strimzi
+# Description: Deploys an Apache Kafka Cluster
+# Required-Vars: STRIMZI_VERSION
+# Var-Defaults: STRIMZI_VERSION=#{STRIMZI_VERSION},PROJECT=myproject
```

Comment:
You have to provide a default version in the `Var-Defaults`, the idea is that the users do not have to pass in a version, if they want to stay with the default one, otherwise if you want the user to every time pass in the version than remove `STRIMZI_VERSION` from `Var-Defaults`